packages/flutter/lib/src/material/tabs.dart

```diff
@@ -70,6 +69,7 @@ class Tab extends StatelessWidget implements PreferredSizeWidget{
     this.text,
     this.icon,
     this.iconMargin = const EdgeInsets.only(bottom: 10.0),
+    this.kTextAndIconTabHeight = 72,
```

Comment:
I am not sure I understand this parameter, what is this to mean? the `k` usually indicates a constant in the framework, so exposing it as a parameter is probably not what we want here.

Reply:
@Piinks changed both the file and the test